packages/web-frontend/src/containers/MapOverlayBar/Control.js

```diff
@@ -96,11 +145,25 @@ export const Control = ({
     }
   }, [isExpanded, setIsExpanded]);
 
+  const reorderedSelectedMapOverlays = useMemo(() => {
+    const results = [];
+    if (!pinnedOverlay) {
+      return selectedMapOverlays;
+    }
+
+    selectedMapOverlays.forEach(overlay => {
+      if (overlay.mapOverlayCode === pinnedOverlay) {
+        results.unshift(overlay);
+      } else results.push(overlay);
+    });
+
+    return results;
+  }, [selectedMapOverlays, pinnedOverlay]);
+
   return (
     <Container>
       <Header>
         <Wrapper>
```

Comment:
We should be able to remove the `Wrapper` component now that `LayersIcon` is gone and just add align-center to the `Header`.